app/views/common_attributes/_common_attribute.html.erb

```diff
@@ -16,8 +16,14 @@
     <% end %>
   </td>
   <td>
-    <%= link_to common_attribute_path(common_attribute), 'aria-label': "delete_#{common_attribute.id}", data: { turbo_confirm: "Are you sure you want to delete this common attribute?" , turbo_method: :delete } do %>
-      <i class="bi bi-trash-fill text-danger"></i>
+    <% if common_attribute.archived %>
```

Comment:
I like that there is one method to delete/archive but I will display different icons and change unarchive button to make it consistent with buildings:
```
<% if common_attribute.archived %>
  <%= link_to unarchive_common_attribute_path(common_attribute), 'aria-label': "unarchive_#{common_attribute.id}" do %>
    <i class="bi bi-archive text-success"></i>
  <% end %>
<% else %>
   <% if common_attribute.has_state? %>
      <%= link_to common_attribute_path(common_attribute), 'aria-label': "delete_#{common_attribute.id}", data: { turbo_confirm: "Are you sure you want to archive this common attribute?" , turbo_method: :delete } do %>
         <i class="bi bi-archive-fill text-success"></i>
      <% end %>
    <% else %>
        <%= link_to common_attribute_path(common_attribute), 'aria-label': "delete_#{common_attribute.id}", data: { turbo_confirm: "Are you sure you want to delete this common attribute?" , turbo_method: :delete } do %>
           <i class="bi bi-trash-fill text-danger"></i>
        <% end %>
   <% end %>
<% end %>
```